contracts/strategies/qv-base/QVBaseStrategy.sol

```diff
@@ -257,14 +257,14 @@ abstract contract QVBaseStrategy is BaseStrategy {
         onlyPoolManager(msg.sender)
         onlyActiveRegistration
     {
-        uint256 recipientLength = _recipientIds.length;
-        if (recipientLength != _recipientStatuses.length) {
-            revert INVALID();
-        }
+        // make sure the arrays are the same length
+        if (_recipientIds.length != _recipientStatuses.length) revert INVALID();
 
-        for (uint256 i; i < recipientLength;) {
+        for (uint256 i; i < _recipientIds.length;) {
```

Comment:
this change increases gas as the .length would have to be computed at ever iteration